docs/documentation/concepts/things.md

```diff
@@ -34,6 +34,16 @@ The statuses UNINITIALIZED, INITIALIZING and REMOVING are set by the framework,
 
 Additionally, the REMOVED state is set by the binding to indicate that the removal process has been completed, i.e. the thing must have been in REMOVING state before.
 
+Furthermore bindings can specify a localized description of the thing status by providing the reference of the localization string, e.g &#64;text/rate_limit. The corresponding handler is able to provide placeholder values as a JSON-serialized array of strings:
```

Comment:
This section looks broken on https://github.com/qivicon/smarthome/blob/744bdc7136123a4c1b4b26235d05fd0c5d60f8f3/docs/documentation/concepts/things.md.

I would also think that this is no content that should go into the "concepts" section.
Imho the better place would be https://github.com/qivicon/smarthome/blob/744bdc7136123a4c1b4b26235d05fd0c5d60f8f3/docs/documentation/development/bindings/thing-handler.md.